plugins/beam-code-completion-plugin/src/main/java/BeamCompletionContributor.java

```diff
@@ -89,8 +98,36 @@ public boolean accepts(@NotNull PsiMethodCallExpression psiMethodCallExpression,
     @Override
     public void fillCompletionVariants(@NotNull final CompletionParameters parameters, @NotNull CompletionResultSet result) {
         if (AFTER_METHOD_CALL_PATTERN.accepts(parameters.getPosition())){
-            for (String transform: beamJavaSDKTransforms) {
-                result.addElement(LookupElementBuilder.create(transform).appendTailText(" org.apache.beam.sdk.transforms", true));
+            try {
+                int offset = parameters.getOffset();
+                String full_code = parameters.getOriginalFile().getText();
+                String code_to_complete = full_code.substring(0, offset);
+
+                Parameters model_parameters = new Parameters(0.1F, false);
+                RequestBody req_body_obj = new RequestBody(code_to_complete, model_parameters);
+
+                String request_body_json = new Gson().toJson(req_body_obj);
+                String model_url = "https://api-inference.huggingface.co/models/bigcode/starcoder";
```

Comment:
lets make this configurable with a default value of `https://api-inference.huggingface.co/models/bigcode/starcoder-base`